scripts/DDL.sql

```diff
@@ -41,6 +42,12 @@ CREATE TABLE jobs.departments(
 	UNIQUE (id, branch_id)
 );
 
+CREATE TABLE jobs.department_head(
+	user_id text REFERENCES jobs.users(upn) ON DELETE CASCADE,
+	department_id int REFERENCES jobs.departments(id) ON DELETE CASCADE,
+	UNIQUE (user_id, department_id)
```

Comment:
💯 on that `UNIQUE`. For future reference... if a `Rahan` would like to view, he can also be defined in `branch_head`, having admin access to all of his `departments`